### ISSUE TITLE
Add support for Hindi

### ISSUE THREAD

**Issue by ghost** [allegro/tradukisto#108]
Tradukisto is the library what converts numbers to human-readable words - for example from `14` to `fourteen`. This issue was created to add support for counting in new language. Depending on how numeral system in your language is similar to other languages (english, arabic, german, slavic) and how many exceptions it have - it can be easy or more demanding. 

Implementing [unit tests](https://github.com/allegro/tradukisto/blob/master/src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/english/EnglishValuesTest.groovy.) will be really helpful for you, I encourage you to create them first (TDD paradigm), before developing new changes in the code.  You can run them in terminal with `./gradlew test` or green button in Intellij code editor.

New language needs to have implementation for [Container](https://github.com/allegro/tradukisto/blob/54e45ee94e13aae69aa64e01d934425c8bd0ed01/src/main/java/pl/allegro/finance/tradukisto/internal/Container.java#L119). Depending on how your language works, having conjugation, gender forms, custom chunking (typically divider is for 3 numbers) etc it may be needed to define more advanced container with custom implementations for counting, for example [turkish](https://github.com/allegro/tradukisto/blob/master/src/main/java/pl/allegro/finance/tradukisto/internal/Container.java#L178). Tests will really be helpful for you to see if it needs custom implementations or not.

Also define `Values` - [translations for base numers](https://github.com/allegro/tradukisto/blob/54e45ee94e13aae69aa64e01d934425c8bd0ed01/src/main/java/pl/allegro/finance/tradukisto/internal/languages/english/EnglishValues.java#L17)  and for [plural forms (thousands, millions etc)](https://github.com/allegro/tradukisto/blob/54e45ee94e13aae69aa64e01d934425c8bd0ed01/src/main/java/pl/allegro/finance/tradukisto/internal/languages/english/EnglishValues.java#L60).
Feel free to ask if you have more questions, we will try to help you. 
https://github.com/allegro/tradukisto/wiki/How-to-start-contributing

**Comment by ronisarkarexe** [comment]
@jglaszka Hey, I saw my language ( Bangla ) missing from the list, can I contribute to the change?

**Comment by jglaszka** [comment]
> @jglaszka Hey, I saw my language ( Bangla ) missing from the list, can I contribute to the change?

sure, we are open for new languages :)

**Comment by ronisarkarexe** [comment]
Can you guide me to contribute? I am new to the open-source community. and I am so excited to work with you.

**Comment by jglaszka** [comment]
Yes. What type of help do you need? You can also check guide first: https://github.com/allegro/tradukisto/wiki/Create-your-first-Pull-Request.

**Comment by ronisarkarexe** [comment]
Question:-
1. how can I run this project locally?
2. What is the name of your branch mean? example my language like ( language_bangla )
3. what is Add label?

**Comment by jglaszka** [comment]
1. How can I run this project locally?
This is an library. There are 2 ways to run this project.
    1. You can "run" it using unit tests (example test: https://github.com/allegro/tradukisto/blob/master/src/test/groovy/pl/allegro/finance/tradukisto/ValueConvertersTest.groovy#L116, how does tests works: https://www.youtube.com/watch?v=o5pE7L2tVV8&ab_channel=BrianFraser) 
    2. Create your own app in Java and import tradukisto (example: https://github.com/allegro/tradukisto/wiki/Getting-Started-with-Tradukisto!).
 2. You can create any name of the git branch as you want, for example language_bangla_implementation
 3. Label can be added in github:
<img width="428" alt="Screenshot 2023-08-22 at 11 03 04 AM" src="https://github.com/allegro/tradukisto/assets/85160468/58cc9ad8-d259-462a-a98e-0db5f963ed55">

**Comment by ronisarkarexe** [comment]
@jglaszka thanks for the guidance. I created a pull request Please check it.

**Comment by girdharsourabh** [comment]
@jglaszka Can you assign this issue to me, I will add Hindi Support.

**Comment by girdharsourabh** [comment]
@jglaszka I have written all implementation for hindi language, only one catch left - "In Hindi, its two digits as separator after thousand and not like English where its always 3 digits between denomination change".  Its like "xx,xx,xx,xxx.yy" format Trying to figure out that override functionality.

@ronisarkarexe Its same in bengali as well, I guess you missed that functionality.

**Comment by jglaszka** [comment]
@girdharsourabh Today I will take a look at it.

**Comment by jglaszka** [comment]
@girdharsourabh I reviewed your [PR](https://github.com/allegro/tradukisto/pull/112) and added a few comments.
